proposed/cache-meta.md

```diff
@@ -279,16 +279,19 @@ do so.
 5. People
 ---------
 
-### 5.1 Editor
+### 5.1 Coordinator
+* Paul Dragoonis
```

Comment:
No other spec lists the Coordinator separately.  It just lists the Sponsors and Coordinator is marked, as below.